docs/user/security/audit-logging.asciidoc

```diff
@@ -14,16 +14,24 @@ by cluster-wide privileges. For more information on enabling audit logging in
 
 [IMPORTANT]
 ============================================================================
+Kibana offers two audit logs: a **deprecated** legacy audit logger, and a new
+ECS-compliant audit logger. We strongly advise using the <<xpack-security-ecs-audit-logging, ECS audit logger>>,
+as the legacy audit logger will be removed in an upcoming version.
+============================================================================
```

Comment:
I think this is fine as-is, we don't need to add the "deprecated" keyword in two different places, but curious to hear what @gchaps thinks.

Reply:
I don't think it hurts to have it in both places.